### PR TITLE
Feature/unittest

### PR DESCRIPTION
**Pull request by jianmingtu**
unit tests

### CODE REVIEW

File: icon-two-common-application-1-1/src/main/java/ca/bc/gov/open/icon/controllers/HealthController.java [hunk 1]
```diff
@@ -35,6 +36,7 @@
 import org.springframework.ws.server.endpoint.annotation.RequestPayload;
 import org.springframework.ws.server.endpoint.annotation.ResponsePayload;
 
+@Profile("!test")
```
Comment (sylar-zeyangz) [inline icon-two-common-application-1-1/src/main/java/ca/bc/gov/open/icon/controllers/HealthController.java:39]: what's this for?

File: icon-two-consumer-models/pom.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>icon-two-consumer-models</artifactId>
-
+    <version>1.0.0</version>
```
Comment (sylar-zeyangz) [inline icon-two-consumer-models/pom.xml:13]: is this needed?

File: icon-two-common-application-1-1/src/main/java/ca/bc/gov/open/icon/controllers/HealthController.java (hunk 1 above)
Comment (jianmingtu) [reply icon-two-common-application-1-1/src/main/java/ca/bc/gov/open/icon/controllers/HealthController.java:39]: without that, the RabbitMQ will try to do a connection and throw error in test.  I do not know exactly how it works, I will ask Ethan as he asked me to add it.

File: icon-two-consumer-models/pom.xml (hunk 2 above)
Comment (jianmingtu) [reply icon-two-consumer-models/pom.xml:13]: yes, it needs to stop connecting RabbitMQ in unit test

**Review by sylar-zeyangz — APPROVED** [review]

**Comment by sylar-zeyangz** [comment]
Please make sure your PR is merged after the completion of review

**Pull request merged**